### PR TITLE
playbook default location

### PR DESCRIPTION
**Pull request by thorinaboenke**
This PR solves issue #64 

- check if the playbook-file is actually a valid filepath and use that
- if not, check if the playbook is inside the current directory
- if not, check if the playbook is in /etc/attackmate/playbooks

- adapt docs
- add unit tests for parse_playbook

image

### CODE REVIEW

**Comment by whotwagner** [comment]
I suggest:

1. check if the playbook-file is actually a valid filepath and use that
2. if not, check if the playbook is inside the current directory
3. check if the playbook is in /etc/attackmate/playbooks

**Comment by thorinaboenke** [comment]
implemented as Hoti suggested, lets not overcomplicate it :)

File: src/attackmate/__main__.py [hunk 1]
```diff
@@ -137,38 +136,52 @@ def parse_playbook(playbook_file: str, logger: logging.Logger) -> Playbook:
...
+    # 3 # Check default playbook directory
+    elif (default_playbook_location / playbook_file_path).exists():
+        target_file = default_playbook_location / playbook_file_path
+
+    else:
+        logger.error(
+            f"Error: Playbook file not found under '{playbook_file_path}' or in the current directory or in /etc/attackmate/playbooks"
+        )
+        exit(1)
+
```
Comment (annaerdi) [inline src/attackmate/__main__.py:165]: Maybe we could add here `logger.info(f"Loading playbook file {target_file}")`
(could be helpful to know which one is used in the end, if there are files with the same name at more than one of those locations)

Comment (whotwagner) [reply src/attackmate/__main__.py:165]: Good idea @annaerdi 
Maybe with logger.debug, so that we are able to debug it if we need to debug the location

**Review by whotwagner — CHANGES_REQUESTED** [review]

File: src/attackmate/__main__.py (hunk 1 above)
Comment (thorinaboenke) [reply src/attackmate/__main__.py:168]: thanks! added the debug logging for target_file

**Review by whotwagner — APPROVED** [review]

**Pull request merged**